model/task/task.go

```diff
@@ -1969,7 +1969,8 @@ func FindRunnable(distroID string, removeDeps bool) ([]Task, error) {
 
 	filterDisabledProjects := bson.M{
 		"$match": bson.M{
-			"project_ref.0." + "enabled": true,
+			"project_ref.0." + "enabled":              true,
```

Comment:
nit: should this use bsonutil.GetDottedKeyName?

Reply:
done